### DISCUSSION TITLE
keyboard layout en-US on the installed system however installation in French

### DISCUSSION THREAD

**Discussion by DidierSpaier** [rhinstaller/anaconda-webui#39] [General]
This is problematic as the password for the first registered  user  is not displayed during initial configuration after installation. Maybe in the definitive version it will be possible to choose the keyboard layout during installation?

Link to quick try in a Qemu VM, FWIW: https://www.freelists.org/post/orca/future-orca-version,2

Cheers,
Didier

**Comment by jkonecny12** [comment]
The keyboard layout configuration is not yet resolved. Thanks for your use-case information!

**Comment by KKoukiou** [comment]
Keyboard configuration was added in Fedora 42.
